frontend/src/components/course/course.tsx

```diff
@@ -70,8 +70,15 @@ const Course: FC = () => {
     setChooseMentorModalOpen((prev) => !prev);
   };
 
-  const handleMentorSelectClick = (mentorId: number): void => {
-    dispatch(courseActions.chooseMentor({ id: mentorId }));
+  const handleMentorSelectClick = (
+    mentorId: number,
+    evt: React.MouseEvent,
```

Comment:
```suggestion
```
now you need to remove it from all places

Reply:
yeah, i see